### PR TITLE
Multi-tag pnp in robot code

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: photon-lib/src/main/java/org/photonvision/PhotonCamera.java [hunk 1]
```diff
@@ -62,6 +69,8 @@ public class PhotonCamera {
     IntegerEntry inputSaveImgEntry, outputSaveImgEntry;
     IntegerEntry pipelineIndexEntry, ledModeEntry;
     IntegerSubscriber heartbeatEntry;
+    private DoubleArraySubscriber cameraIntrinsicsEntry;
```
Comment (mcm001) [inline photon-lib/src/main/java/org/photonvision/PhotonCamera.java:72]: nit: should be named Sub or Subscriber not Entry

File: photon-lib/src/main/java/org/photonvision/PhotonCamera.java [hunk 2]
```diff
@@ -298,6 +312,20 @@ public boolean isConnected() {
         return (now - prevHeartbeatChangeTime) < HEARBEAT_DEBOUNCE_SEC;
     }
 
+    public Optional<Matrix<N3, N3>> getCameraMatrix() {
+        var cameraMatrix = cameraIntrinsicsEntry.get();
+        if (cameraMatrix != null) {
+            return Optional.of(new MatBuilder<>(Nat.N3(), Nat.N3()).fill(cameraMatrix));
```
Comment (mcm001) [inline photon-lib/src/main/java/org/photonvision/PhotonCamera.java:318]: assert length of array here, too?

File: photon-lib/src/main/java/org/photonvision/SimPhotonCamera.java [hunk 3]
```diff
@@ -33,6 +33,7 @@
 import org.photonvision.targeting.PhotonTrackedTarget;
 
 @SuppressWarnings("unused")
+// TODO: Add method to set camera intr/extrinsics
```
Comment (mcm001) [inline photon-lib/src/main/java/org/photonvision/SimPhotonCamera.java:36]: reminder lol

**Comment by MrRedness** [comment]
Just curious, is there a reason this is being done in robot code and not something toggled on the co-processor?

File: photon-lib/src/main/java/org/photonvision/SimPhotonCamera.java [hunk 4]
```diff
@@ -33,6 +37,7 @@
...
```
Comment (Bankst) [inline photon-lib/src/main/java/org/photonvision/SimPhotonCamera.java:40]: looks done to me

**Review by Bankst — DISMISSED** [review]

**Review by Bankst — APPROVED** [review]

**Pull request merged**